### ISSUE TITLE
[Documentation request] Guide for tests

### ISSUE THREAD

**Issue by QuantumInformation** [GetBlimp/ember-parse#1]
I notice that we can run the dummy app and make changes to the live database. I was wondering if we could agree on a future proof testing approach. Is there a way to create a sandbox environment on parse that is set to a known state each time tests are run?

Note I tried running the tests and get this:
![image](https://cloud.githubusercontent.com/assets/216566/8020118/f8ae6472-0c75-11e5-9119-fc3ee27c1eb8.png)

**Comment by gcollazo** [comment]
I've been thinking for some time about how to do tests and I think that a good starting point would be to test agains the official parse javascript library.

As for testing against real parse servers, what we could do is create a cloud code function that resets all data on the parse side before running tests.

**Comment by QuantumInformation** [comment]
Good idea in regards to cloud code. We could create a folder inside this
repo for the CC project too.

-- 

Kind regards
Nikos
